app/assets/stylesheets/modules/form.scss

```diff
@@ -0,0 +1,20 @@
+@import "variables";
+@import "mixins";
+
+.sign-up {
+  width: 50%;
+  margin-left: auto;
+  margin-right: auto;
+
+  .legend{
+    font-family:$dogstay-font-theme;
+    font-size: 2em;
+  }
+
+  .postfix {
+    width:30%;
+    margin-left: auto;
+    margin-right: auto;
+    background: teal;
```

Comment:
Color `teal` should be written in hexadecimal form as `#008080`<br>Color literals like `teal` should only be used in variable declarations; they should be referred to via variable everywhere else.